src/runtime_src/core/include/experimental/xrt_graph.h

```diff
@@ -182,4 +381,10 @@ xrtGraphUpdateRTP(xrtGraphHandle gh, const char *hierPathPort, const char *buffe
 int
 xrtGraphReadRTP(xrtGraphHandle gh, const char *hierPathPort, char *buffer, size_t size);
 
+/// @endcond
+
+#ifdef __cplusplus
+}
+#endif
+
```

Comment:
For the API's for user, is it good to keep consistence between C version and C++ version?
xrtGraphOpen  ->  open  //May have
xrtGraphClose ->  close
xrtGraphReset ->  reset
xrtGraphTimeStamp -> timeStamp
xrtGraphRun -> run
xrtGraphWaitDone -> waitDone
xrtGraphWait -> wait
xrtGraphSuspend -> suspend
xrtGraphResume -> resume
xrtGraphEnd -> end
xrtGraphUpdateRTP -> updateRTP
xrtGraphReadRTP -> readRTP

Reply:
Who is your typical ser who will use this API both in C and in C++ at the same time?
Even just 1 user using just 1 API seems already rare and anyway will require to look at the documentation.
For me, I will never use the C API, maybe the C++ API...
So, yes, improve the C++ API. :-)